reinstall-cli/src/main.rs

```diff
@@ -0,0 +1,84 @@
+//! The main entrypoint for bootc-reinstall
+
+use std::{io::Write, process::Command};
+
+use anyhow::{Context, Result};
+
+fn run() -> Result<()> {
+    bootc_lib::cli::tracing_util::initialize_tracing();
+
+    tracing::trace!("starting bootc-reinstall");
+
+    prompt()?;
+
+    let command_and_args = [
+        // Rootless is not supported
+        "sudo",
```

Comment:
I personally get a bit weirded out when tools invoke `sudo` automatically. It's not wrong per se, but it can lead to confusion. Do you think we really need it?

Reply:
100% definitely, just copy pasted the command without giving it much thought. The user should do the sudo themselves